src/pages/setting/SettingPage.tsx

```diff
@@ -15,35 +27,283 @@ const NavigationLayout = ({ children }: NavLayoutProps) => {
       <TopNavigation
         _TopBar={
           <TopNavigation.TopBar
-            centerContent={
-              <TopNavigation.TopBar.CenterTitle>내 정보</TopNavigation.TopBar.CenterTitle>
-            }
-            rightContent={
-              <TopNavigation.TopBar.CloseButton
+            leftContent={
+              <TopBar.PrevButton
                 onClick={() => {
                   navigate(-1);
                 }}
               />
             }
+            centerContent={<div>내 정보</div>}
           />
-        }></TopNavigation>
+        }
+      />
       {children}
+      <BottomNavigation />
     </>
   );
 };
 
 // #20240429.syjang, 환경설정 테스트 페이지입니다. 추후 테마 변경 시 아래와 같이 가져다 쓰면 됩니다.
 const SettingPage = () => {
+  const navigate = useNavigate();
+  const [budget, setBudget] = useState<Budget>();
+  const [genderColor, setGenderColor] = useState<Gender>(null);
+  const [selections, setSelections] = useState({
+    EI: '',
+    NS: '',
+    TF: '',
+    JP: '',
+  });
+
+  const { setLogoutState } = useAuthStore((state) => {
+    return { setLogoutState: state.setLogoutState };
+  });
+
+  const handleGenderColor = (gender: Gender) => {
+    setGenderColor(gender);
+  };
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setBudget(e.target.value);
```

Comment:
```suggestion
    setBudget(parseInt(e.target.value));
```

이런식으로 하면 굳이 Budget에 string, number 타입 두개를 지정할 필요가 없을 것 같습니다.